lib/cucumber/cli/options.rb

```diff
@@ -464,6 +465,31 @@ def extract_environment_variables
         end
       end
 
+      def adjust_line_numbers
+        @args.map! do |arg|
+          next arg unless arg =~ /:\d+$/
+          line_numbers = arg.split(':')
+          file = line_numbers.shift
+          lines = File.readlines(file)
+          line_numbers.map! do |line_number|
+            line_number.to_i.downto(1).find do |i|
+              line = lines[i - 1]
+              raise ScenarioOutlineFound if line =~ /^\s+Scenario Outline:/
```

Comment:
In Gherkin today also Scenarios can have an Examples Table ("Scenario Outline" only a synonym for "Scenario"), so it is not possible to decide if it is a simple Scenario (without Examples Table) of a Scenario with examples (previously known as "Scenario Outline" only by examine the keyword for the scenario line.